src/peoplemeasurement/models.py

```diff
@@ -60,3 +60,17 @@ class VoorspelIntercept(models.Model):
     sensor = models.CharField(max_length=255)
     toepassings_kwartier_volgnummer = models.IntegerField()
     intercept_waarde = models.FloatField()
+
+
+class Area(models.Model):
+    sensor = models.CharField(max_length=255)
+    name = models.CharField(max_length=255)  # Naam van meetgebied
+    geom = models.PolygonField()  # Polygoon dat het meetgebied omvat
+    area = models.IntegerField()  # Oppervlakte van het meetgebied in m2
+
+
+class Line(models.Model):
+    sensor = models.CharField(max_length=255)
```

Comment:
sensoren worden nu overal uitgeschreven. In waarnemingen-voertuigen werd dat uiteindelijk een probleem vanwege de hoeveelheid data. Is het handig hier een Sensor model te maken en naar te verwijzen?